### ISSUE TITLE
Define the New Video Screen

### ISSUE THREAD

**Issue by internetmosquito** [wizbots/labtab#14]
We need to define the mockup for when users of labtab click on create video.

Basically we need a screen where they can input the following info:

1.- Kids -> If we're coming from the lab detail view after selecting kid/s, this should be automatically populated, if not, a list of kids in the selected Lab, if no Lab is selected, nothing should be show here
2.- Project -> A dropdown with all the available projects
3.- Category -> Challenge, Game or Contraption
4.- Knowledge nuggets -> I think this would be great if we could filter based on the project and level of kids, but for now, whole list of KN
5.- Description

**Comment by solnishka** [comment]
@internetmosquito , what do you think if we leave the same algorythm as it is on wizbots.com:
![untitled-5](https://cloud.githubusercontent.com/assets/9883696/20246286/c57e9412-a9c4-11e6-9058-dfed01e92194.jpg)

or here: https://projects.invisionapp.com/d/main#/console/9523180/204776732/preview

when we start download video from students selection they are automatically added to edit screen.

**Comment by internetmosquito** [comment]
@solnishka I don't follow Tanya, the process is quite different in Labtab, I have to confirm this, but I don't think we need to do as we do now in Wizbots where:

1.- Mentors upload a video that is not compatible with most modern browsers
2.- Video is first uploaded and dealt with via Amazon so it gets transcoded to a more compaible format
3.- When video is uploaded to Wonderwall then Mentor gets notified and can change video data

Also take into account that the whole idea is that we can't guarantee that we have internet connectivity, thus this dialog that requires to upload a video first before being able to edit video data is not going to work. We need to be able to edit the video data directly, something closer to:

![selection_139](https://cloud.githubusercontent.com/assets/1500392/20246540/fa52c840-a9b8-11e6-9c72-845b9609e7e1.png)

Makes sense?

**Comment by solnishka** [comment]
@internetmosquito , then may be we'll have the same pages for add video and edit video?
Just when you come to add new video you see the button Upload except the video screen, all the rest is the same, what do you think?

**Comment by internetmosquito** [comment]
Yeah, that's what I said earlier, new video and edit video should be very similar, but new video does not have the preview thumnail...for obvious reasons, but we can keep it if it saves time, I don't mind, let's see what @kapoorutd has to say about thi

**Comment by kapoorutd** [comment]
As per me only saved video on Device would have thumbnail otherwise need to
do a lot to create thumbnail for new one.
Also editing video and merging should be separate page. Let me add a
screenshot as we would be working against library and would be difficult to
modify that.

**Comment by internetmosquito** [comment]
@kapoorutd what I meant was, when we create a new video we don't have the preview because file does not exist...does it make sense to have same view as edit video but with thumbnail empty or showing something like "Preview not available" or do we need 2 windows? One for new another for edit? The only difference would be the video thumbnail

**Comment by solnishka** [comment]
@internetmosquito , I propose something like that:
![18 111](https://cloud.githubusercontent.com/assets/9883696/20437530/5cdc4792-adc5-11e6-952f-987d3aed1f00.png)

**Comment by internetmosquito** [comment]
@solnishka looks good, but a few details:

1.- Remove the "Please upload your video file" and Browse button -> Change it for No Preview available, we're going to create the video from this screen so nothing can be uploaded...

2.- Please remove the "Copy Link" and "Select as pick of the week" buttons

3.- Change the Update Project button to -> Create -> This should take to video recording screen @kapoorutd confirm this please

4.- Add a cancel button that takes a back

@kapoorutd please give feedback on how hard is to do this on Android

**Comment by solnishka** [comment]
![21 11](https://cloud.githubusercontent.com/assets/9883696/20478536/96192516-afeb-11e6-9e19-f7020a594bc1.png)

**Comment by internetmosquito** [comment]
@solnishka looks good, thanks!
@kapoorutd if this is ok with you, we move it to InVisio

**Comment by kapoorutd** [comment]
Looks good.
